src/box/lua/config/utils/funcutils.lua

```diff
@@ -0,0 +1,21 @@
+local methods = {}
```

Comment:
Sorry for nitpicking, but I'm trying to use the term 'method' in context of Lua, when referring to a function that accepts a particular object as the first argument (`self`), and this function stored inside a table or its `__index` table that allows to call it using a colon (`obj:mymethod(<...>)`).

There are several frequent styles how a Lua module is organized.

```lua
local M = {}

function M.foo(<...>)
    <...>
end

return M
```

Here `M` stands for 'module', not 'methods'. At least, in my understanding.

Another valid style:

```lua
local mymodule = {}

function mymodule.foo(<...>)
    <...>
end

return mymodule
```

And the third one:

```lua
local function foo(<...>)
    <...>
end

return {
    foo = foo,
}
```

I have no strict preference here. If all the other modules in this directory use the same style, let's use it here as well. If not, choose any of them.

But I think that 'methods' doesn't fit here good.

Reply:
> Sorry for nitpicking

No problem, I refactored it to use the third option.